MAUI/PDF-Viewer/Signature.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: post
+title: Electronic Signature in .NET MAUI PDF Viewer control | Syncfusion
+description: Learn here all about Electronic Signature in Syncfusion .NET MAUI PDF Viewer (SfPdfViewer) control and its types.
+platform: MAUI
+control: SfPdfViewer
+documentation: ug
+---
+
+# Electronic Signature in .NET MAUI PDF Viewer (SfPdfViewer)
+
+The electronic signature feature of [SfPdfViewer](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.PdfViewer.SfPdfViewer.html) allows you to add, remove signature in the PDF document. This section will go through the various types and functions available in PDF Viewer for working with signature.
+
+## Types of electronic signature
+
+The following signature types are now available in the PDF Viewer.
+
+1.	Handwritten signature.
+2.	Text signature.
+3.  Image signature.
+
+## Add signature by UI interaction
+
+This section will go through how to add signature by UI interaction to a PDF document.
+
+You can add signature to a PDF document with UI interaction using signature dialog. The following steps explains how to add signature on a PDF.
+
+	1. Set the AnnotationMode property of the SfPdfViewer to `Signature`. It activates the signature mode on the control and signature dialog box will be open.
+	2. create a signature in any one of the types
+	3. You can customize the color of the signature for handwritten and text signature types
```

Comment:
3. You can customize the color of the signature for handwritten and text signature types.